plugins/bufferlines/barbar.nix

```diff
@@ -126,15 +150,22 @@ in {
       But only a static string is accepted here.
     '';
 
-    # Keybinds concept:
-    # keys = {
-    #   previousBuffer = mkBindDef "normal" "Previous buffer" { action = ":BufferPrevious<CR>"; silent = true; } "<A-,>";
-    #   nextBuffer = mkBindDef "normal" "Next buffer" { action = ":BufferNext<CR>"; silent = true; } "<A-.>";
-    #   movePrevious = mkBindDef "normal" "Re-order to previous" { action = ":BufferMovePrevious<CR>"; silent = true; } "<A-<>";
-    #   moveNext = mkBindDef "normal" "Re-order to next" { action = ":BufferMoveNext<CR>"; silent = true; } "<A->>";
-
-    #   # TODO all the other ones.....
-    # };
+    keymaps =
+      {
+        silent = mkOption {
+          type = types.bool;
+          description = "Wether barbar keymaps should be silent";
+          default = false;
+        };
```

Comment:
It's a mkEnableOption "silent keymaps for barbar"

Reply:
Done :)